### PR TITLE
Add count of matches won, lost, and total to Stats page

### PR DESCRIPTION
**Pull request by amina-alibasic**
Added `(X Won, X Lost, X Total)` text to the stats page, next to the win rate percentage. Updated tests accordingly.

### CODE REVIEW

File: client/js/components/StatsLeaderboardCell.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
         </v-list>
         <h3>Win Rate</h3>
         <v-list :data-win-rate="`${username}-week-${week}`">
-          {{ winRatePercentage }}
+          {{ winRatePercentage }} {{ winRateText }}
```
Comment (itsalaidbacklife) [inline client/js/components/StatsLeaderboardCell.vue:31]: Let's combine these into one higher level computed prop that concatenates them

File: client/js/components/StatsLeaderboardCell.vue [hunk 2]
```diff
@@ -154,6 +154,12 @@ export default {
       const winRate = Math.floor((this.wins / this.weekCount) * 100);
       return `${winRate}%`;
     },
+    losses() {
+      return this.weekCount - this.wins;
+    },
+    winRateText() {
+      return `(${this.wins} Won, ${this.losses} Lost, ${this.weekCount} Total)`;
+    },
```
Comment (itsalaidbacklife) [inline client/js/components/StatsLeaderboardCell.vue:162]: Let's include the WinRatePercentage in this and then the template above can just display the winRateText

**Review by itsalaidbacklife — APPROVED** [review]
Looks good. Requesting a small refactoring

File: client/js/components/StatsLeaderboardCell.vue (hunk 1 above)
Comment (amina-alibasic) [reply client/js/components/StatsLeaderboardCell.vue]: Done

File: client/js/components/StatsLeaderboardCell.vue (hunk 2 above)
Comment (amina-alibasic) [reply client/js/components/StatsLeaderboardCell.vue]: Done

**Pull request merged**